C/factors.C

```diff
@@ -2,15 +2,14 @@
 #include<conio.h>
```

Comment:
- You can improve this file more by assigning proper names to variables instead of a,b,i,j.
- add comments to explain the logic of the code
- rename the file to a better name

Reply:
@kaustubhgupta done, review!